README.md

```diff
@@ -144,7 +144,8 @@ This test suite expects following environment variables set:
  - `STORAGE_API_MAINTENANCE_URL` - URL for maintenance testing (https://maintenance-testing.keboola.com/)
  - `STORAGE_API_TOKEN_IN_SAME_ORGANIZATION` - Storage API token associated to project in the same organization as `STORAGE_API_TOKEN`.
  - `STORAGE_API_TOKEN_IN_OTHER_ORGANIZATION` - Storage API token associated to project in the other organization.
-
+ - `STORAGE_API_TOKEN_USER_WITH_OTHER_EMAIL` - Storage API token associated to user with other email and in the same organization as `STORAGE_API_TOKEN`.
+ - `STORAGE_API_TOKEN_USER_EMAIL_IN_OTHER_ORG` - Storage API token associated to user in the other organization.
```

Comment:
Ja bych nove tokeny nezavadel.
Stacilo by jen upravit popis toho co vyzaduji ty tokeny `STORAGE_API_TOKEN_IN_SAME_ORGANIZATION`, `STORAGE_API_TOKEN_IN_OTHER_ORGANIZATION` ktere jsi pridaval minule ne?

Tim by se zjednodusil setup.

Reply:
Jo mas pravdu, oddelal som ich a ten jeden som upravil aby musel mat iny email pretoze je to potreba v tychto testoch